### PR TITLE
Lab/modal flow

### PR DESCRIPTION
**Pull request by tesk9**
Part of https://www.pivotaltracker.com/story/show/164609653

Spec is here: https://paper.dropbox.com/doc/Product-spec-new-intro-modal-for-Self-Review--AaqPb8vfJfYBB82kmLvhUOcbAg-HJSZsPmnjm8drCdxuWdLm

slide modal pre slide

This sets up everything but the actual "Slide" -- it doesn't animate in yet.

### CODE REVIEW

File: src/Nri/Ui/SlideModal/V1.elm [hunk 1]
```diff
@@ -0,0 +1,308 @@
...
+        , Css.property "box-shadow" "0 1px 10px 0 rgba(0, 0, 0, 0.35)"
+        , Css.displayFlex
+        , Css.alignItems Css.center
+        , Css.flexDirection Css.column
+        , Css.flexWrap Css.noWrap
+        , Fonts.baseFont
+        ]
+        [ Role.dialog
+        , Widget.modal True
+        , labelledBy labelledById
```
Comment (juanedi) [inline src/Nri/Ui/SlideModal/V1.elm]: today I learned about this attribute! 😮

File: src/Nri/Ui/SlideModal/V1.elm [hunk 2]
```diff
@@ -0,0 +1,308 @@
...
+
+
+type Dot
+    = Active
+    | Inactive State String
+    | InactiveDisabled State String
+
+
+dot : Dot -> Html.Html State
+dot type_ =
```
Comment (juanedi) [inline src/Nri/Ui/SlideModal/V1.elm]: 🤔 We use these dots on various places... should we add a PT story/chore to make a separate doodad for these?

Comment (tesk9) [reply src/Nri/Ui/SlideModal/V1.elm]: I'll add a tech debt card, if that works?

Comment (juanedi) [reply src/Nri/Ui/SlideModal/V1.elm]: That would be great. Thank you!

**Pull request merged**